lib/plugins/aws/lib/monitorStack.js

```diff
@@ -74,7 +74,9 @@ module.exports = {
                     }
                     // Keep track of first failed event
                     if (eventStatus
-                      && eventStatus.endsWith('FAILED') && stackLatestError === null) {
+                      && (eventStatus.endsWith('FAILED')
+                      || eventStatus === 'UPDATE_ROLLBACK_IN_PROGRESS')
```

Comment:
I'm not sure if an "in progress" message should be translated to a final error event.
Imo the final message that finalizes the stack rollback should be used to add the failure. I think that is "UPDATE_ROLLBACK_COMPLETE".

The real issue here is, that _.endsWith("_COMPLETE") is treated as success below. So checking for the UPDATE_ROLLBACK_COMPLETE additionally as final error state should be consistent.